chargebee/src/main/java/com/chargebee/android/Chargebee.kt

```diff
@@ -230,4 +236,30 @@ object Chargebee {
             Success(cbTempToken)
         }, completion, logger)
     }
+
+    /**
+     * This method will be used to show the Manage Subscriptions Settings in your App,
+     *
+     * @param [context] Current activity context
+     * @param [productId] Optional. Product Identifier.
+     * @param [packageName] Optional. Application Id.
+     */
+    fun showManageSubscriptionsSettings(
+        context: Context,
+        productId: String = "",
```

Comment:
If `productId` and `packageName` are optional, can we mark them as `?` instead of defaulting to `""`?